REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.4
-Compat 0.7.15
+Compat 0.9.4
```

Comment:
looking at what this package uses compat for, looks like maybe only the show rewriting? https://github.com/JuliaLang/Compat.jl/commit/57122c4d8275f7ba41229c74f01db490311787aa